prow/plugins.yaml

```diff
@@ -35,7 +35,6 @@ approve:
 - repos:
   - kubernetes/charts
   - kubernetes/cluster-registry
-  - kubernetes/community
```

Comment:
Shouldn't you add this to a separate repos block with `implicit_self_approve: false`?

Reply:
false is the default, unless you override it.

Reply:
Ah, the `plugins: - kubernetes/community: - approve` turns on approval tracking.